server.js

```diff
@@ -544,15 +544,17 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 // SonarQube code coverage
-camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/coverage\.(svg|png|gif|jpg|json)$/,
+camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/(.*)\.(svg|png|gif|jpg|json)$/,
     cache(function(data, match, sendBadge, request) {
       var scheme = match[1];
       var serverUrl = match[2];  // eg, `sonar.qatools.ru`.
       var buildType = match[3];  // eg, `ru.yandex.qatools.allure:allure-core:master`.
-      var format = match[4];
+      var metric = match[4];
+      var format = match[5];
+      var realMetric = metric === 'tech_debt' ? 'sqale_debt_ratio' : metric
       var apiUrl = scheme + '://' + serverUrl + '/api/resources?resource=' + buildType
-          + '&depth=0&metrics=coverage&includetrends=true';
-      var badgeData = getBadgeData('coverage', data);
+          + '&depth=0&metrics=' + realMetric + '&includetrends=true';
+      var badgeData = getBadgeData(escapeFormat(metric), data);
```

Comment:
Is there a reason why you used `escapeFormat()`? It was designed specifically to escape raw custom badge information from the URL. I don't think it makes sense here.